src/fontra/views/applicationsettings/panel-base.js

```diff
@@ -0,0 +1,91 @@
+import { UndoStack, reverseUndoRecord } from "../core/font-controller.js";
+import * as html from "../core/html-utils.js";
+import { commandKeyProperty } from "../core/utils.js";
+
+export class BaseInfoPanel {
+  constructor(applicationSettingsController, panelElement) {
+    this.applicationSettingsController = applicationSettingsController;
+    //this.fontController = applicationSettingsController.fontController;
+    this.panelElement = panelElement;
+  }
+
+  visibilityChanged(onOff) {
+    this.visible = onOff;
+    if (onOff && !this.initialized) {
+      this.initializePanel();
+      this.initialized = true;
+    }
+  }
+
+  initializePanel() {
+    this.undoStack = new UndoStack();
+
+    // const subscribePattern = Object.fromEntries(
```

Comment:
These things should be deleted: it is when a view deals with a font. This view doesn't.

Reply:
I removed the commented out code.